backend/database/actionsWithTables/categorieActions.js

```diff
@@ -17,6 +17,17 @@ const getAllCategories = async () => {
     return await categories;
 };
 
+const updateCategorie = async (id, info) => {
+    await Categorie.update(
+        {
+            name: info.modalName,
```

Comment:
Maybe should also add `is_deleted` status updating? For example for restoring entity